internal/sbi/producer/generate_auth_data.go

```diff
@@ -182,7 +182,12 @@ func GenerateAuthDataProcedure(authInfoRequest models.AuthenticationInfoRequest,
 			Detail: err.Error(),
 		}
 
-		logger.UeauLog.Errorln("Return from UDR QueryAuthSubsData error")
+		switch res.StatusCode {
+		case 404:
```

Comment:
It is better to replace `404` with `http.StatusNotFound`.

Reply:
I have fixed.